code/datums/mutations/hulk.dm

```diff
@@ -256,4 +256,17 @@
 	yeeted_person.throw_at(T, 10, 6, the_hulk, TRUE, TRUE)
 	log_combat(the_hulk, yeeted_person, "has thrown by tail")
 
+/datum/mutation/human/hulk/wizardly
+	species_allowed = null //yes skeleton/lizard hulk
```

Comment:
Do you have pictures for what a hulk lizard / hulk skeleton looks like? I'd hope there's still a visual tell they're stun immune outside of the laser eyes

(Yes I know Magus Armor exists)

Reply:
I don't see how it could let you bypass lich revive stun when Hulk doesnt retroactively cleanse stuns afaik

Reply:
![image](https://github.com/tgstation/tgstation/assets/53100513/ec8cbae8-8d05-4550-b8c3-67a14440251e)
![image](https://github.com/tgstation/tgstation/assets/53100513/95890a38-504a-4c01-88f9-14021a5f0fd7)
![image](https://github.com/tgstation/tgstation/assets/53100513/659ef03c-83e7-4165-bd93-783b01f40cda)

Strange, i'll look into skellies

Reply:
Androids, abductors, Hzombies, monkeys, nightmares, all fail - Looks like it breaks on anything that has hardsprited bodyparts. Hrm. How do we solve this? Locking Mutate behind species that get their sprite changed? The only 'proper' fix here would be a full refactor of species, I think?

Reply:
Fixed - races that don't have mutant color or skin tone just get a funny green filter.